_docs_agent/secrets.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: page
+title: Using secrets in configuration
+description: Configure the CoScale agent and plugins using secrets.
+---
+
+When creating your Docker image you can add a label to tell the CoScale agent how to monitor this image when it's detected. To make this work in an environment where you sensitive data is stored in a secret. You can also tell the agent to read usernames and passwords from a secrets file/mount.
```

Comment:
Kunnen we dit samenvoegen met de environment var pagina? Meestal volgt die vraag sowieso en ze tonen bijna exact hetzelfde.

Reply:
Both pages talk about the same use-case, just other ways of managing it. I'd rather not have two pages with almost exactly the same content. 

I think there should also be a section about using the environment variables without labels, as this is also supported for containers and not only packaged agent.

Reply:
@Firewall you want to look at this together?